fetch-api/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@shoutem/js-api",
```

Comment:
Can we rename this to `api-sdk`? `js-api` was a bit confusing to me when I first saw it. It looked like this is some sort of JS interface with other Shoutem components...

We also had a convention to to add `react-` before a package name when it was react specific, so I would call this:
```
api-sdk
react-api-sdk
```

Sorry about this, I know you asked about it a long time ago, but it seemed ok to me at that point.